README.md

```diff
@@ -119,6 +119,8 @@ To work correctly, you will first need:
 
 ## Installing
 
+Now that the requirements are satisfied, you can install Spaceship via any of the following tools.
```

Comment:
```suggestion
Now that the requirements are satisfied, you can install Spaceship ZSH via any of the following tools.
```

Reply:
May be Spaceship _prompt_ ?

Reply:
@salmanulfarzy I used to use _Spaceship ZSH_ as the marketing name of the project. In that case, better to name it _Spaceship ZSH prompt_.

Reply:
Oh.. Okay. Also we mostly use `Zsh` instead of `ZSH` in docs, So may be that ?